lib/Gateway.js

```diff
@@ -1108,65 +1217,65 @@ Gateway.prototype.discoverValue = function (node, valueId) {
           } else {
             // ... otherwise use a light dimmer configuration
             // brightness level
-            const rgb = node.values['51-1-0']
-            if (rgb) {
-              cfg = copy(hassCfg.light_rgb_dimmer)
-              cfg.discovery_payload.rgb_state_topic = this.mqtt.getTopic(
-                this.valueTopic(node, rgb)
-              )
-              cfg.discovery_payload.rgb_command_topic =
-                cfg.discovery_payload.rgb_state_topic + '/set'
-              cfg.discovery_payload.brightness_state_topic = this.mqtt.getTopic(
-                topic
-              )
-              cfg.discovery_payload.brightness_command_topic =
-                cfg.discovery_payload.brightness_state_topic + '/set'
-            } else {
-              cfg = copy(hassCfg.light_dimmer)
-            }
+            // const rgb = node.values['51-1-0']
```

Comment:
again `todo`?

Reply:
yep, It requires https://github.com/zwave-js/node-zwave-js/issues/806

Reply:
added a todo